src/main/java/com/rewedigital/composer/routing/Match.java

```diff
@@ -1,45 +1,55 @@
 package com.rewedigital.composer.routing;
 
+import java.time.Duration;
 import java.util.Objects;
+import java.util.Optional;
 
 public class Match {
 
     private final String backend;
+    private final Optional<Duration> ttl;
     private final RouteTypeName routeType;
 
-    private Match(final String backend, final RouteTypeName routeType) {
+    private Match(final String backend, final Optional<Duration> ttl, final RouteTypeName routeType) {
         this.backend = backend;
+        this.ttl = ttl;
         this.routeType = routeType;
     }
 
-    public static Match of(final String backend, final RouteTypeName routeType) {
-        return new Match(backend, routeType);
+    public static Match of(final String backend, final Optional<Duration> ttl, final RouteTypeName routeType) {
+        return new Match(backend, ttl, routeType);
     }
 
     public String backend() {
         return backend;
     }
 
+    public Optional<Duration> ttl() {
```

Comment:
Might be me not understanding the coverage report, but it looks like this new method is not covered by any test?